js/libs/form.js

```diff
@@ -1276,7 +1277,44 @@ Fliplet().then(function() {
                   // Update live field
                   field.options = options;
                 },
-                instance: field
+                on: function(eventName, fn) {
+                  var eventListeners = data.fieldEventListeners;
+
+                  if (!eventListeners) {
+                    eventListeners = {};
+                  }
+
+                  eventListeners[field.name] = eventListeners[field.name] || {};
+                  eventListeners[field.name][eventName] = eventListeners[field.name][eventName] || [];
+                  eventListeners[field.name][eventName].push(fn);
+
+                  data.fieldEventListeners = eventListeners;
+                },
+                off: function(eventName, fn) {
+                  var eventListeners = data.fieldEventListeners[field.name][eventName];
```

Comment:
`off()` might be executed before `on`, which means `data.fieldEventListeners[field.name][eventName]` will fail due to `data.fieldEventListener` not existing.

I suggest using `_.get(data, ['fieldEventListeners', field.name, eventName])`